roles/ceph-osd/tasks/pre_requisite.yml

```diff
@@ -58,3 +58,23 @@
   when:
     - cephx
     - item.copy_key|bool
+
+- set_fact:
+     osd_uuids: |
+      {%- set o=[] %}
+      {%- for device in  devices %}
+        {%- if o.append( ( inventory_hostname + device + ansible_date_time.iso8601_micro + "osd_uuids") | to_uuid ) %}
```

Comment:
Sorry I meant `/dev/disk/by-id/` not uuid. I'm just not sure to understand the added value of your patch. What's the benefit of registering an UUID ourselve instead of letting ceph-disk creating one? What's the use case?

Thanks!